app/uk/gov/hmrc/vatsubscription/services/VatCustomerDetailsRetrievalService.scala

```diff
@@ -49,6 +49,13 @@ class VatCustomerDetailsRetrievalService @Inject()(vatCustomerDetailsConnector:
     }).value
   }
 
+  def extractWelshIndicator(vrn: String)(implicit hc: HeaderCarrier): Future[Boolean] = {
+    retrieveVatCustomerDetails(vrn).map {
+      case Right(details) => details.welshIndicator.contains(true)
+      case Left(_) => false
```

Comment:
Please put a logger message in here in case of failure. We also probably don't want to default this value, as if the first call failed but the update passed, there's a (probably very small) risk of overwriting the user's welsh indicator flag. We should probably let it fail and report the failure back to the frontend